src/logbooks/logbooks.service.ts

```diff
@@ -44,12 +49,14 @@ export class LogbooksService {
     if (this.logbookEnabled) {
       if (this.username && this.password) {
         try {
-          const accessToken = await this.login(this.username, this.password);
+          const shouldRenewToken = await this.shouldRenewToken();
+          if (shouldRenewToken) {
+            await this.login(this.username, this.password);
+          }
+
           Logger.log("Fetching Logbooks", "LogbooksService.findAll");
           const res = await firstValueFrom(
-            this.httpService.get<Logbook[]>(this.baseUrl + "/Logbooks", {
-              headers: { Authorization: `Bearer ${accessToken.token}` },
-            }),
+            this.httpService.get<Logbook[]>(this.baseUrl + "/Logbooks"),
```

Comment:
why the token was removed?
Does this mean that I can retrieve the logbook without authentication?

Reply:
In the loopback repo we don't use headers to retrieve token. The token used for loopback is created and stored on the first success login to the synapse service, and then will re-use the token until loopback service is restarted.

Reply:
Does this mean that there is no authentication between SciCat BE and loopback connector?
If so, is that what we want to do?

Reply:
we need to provide synapse username & password on the first time login, which is saved in scicat backend .env file.
After retrieved token from first time login,  we use the token only until either scicat BE or loopback application restarts.
One of the main reason we should try to use token exclusively and minimize login attempt is because, synapse service is very sensitive to login Rate limit with same account. If we somehow triggeres login more than 2 times within short period time(less than 1 minute or so), synapse will throw` rate limit excedeed` error and we will need to wait for 1~5 minutes login restriction untill next login attempt.